test/test_dht.cpp

```diff
@@ -3375,6 +3376,44 @@ TORRENT_TEST(compare_ip_cidr)
 #endif
 }
 
+TORRENT_TEST(dht_state)
+{
+	dht_state s;
+
+	s.nid = to_hash("0000000000000000000000000000000000000001");
+	s.nodes.push_back(udp::endpoint(addr("1.1.1.1"), 1));
```

Comment:
there's a utility function called ``ep()``, which makes these kinds of addresses shorter and easier to read